.github/workflows/ocwm-issue-collector.yml

```diff
@@ -0,0 +1,86 @@
+      run: npm install @octokit/core
+    - name: Adding Issues
+      uses: actions/github-script@v6
+      env:
+        PLACEHOLDER: '<!-- | [TOPIC] [IssuePRDiscussion] | [owner] | -->'
+        OWNER: ${{ vars.ORGANISATION }}
+        REPO: 'community'
+        REPO_NAMES: ${{ vars.REPOSITORIES }}
+        LABELS: ${{ vars.LABELS }}
+        MY_TOKEN: ${{ secrets.AUTH_TOKEN }}
+      with:
+        script: |
+          try {         
+            const repoNames = process.env.REPO_NAMES;
+            const labels = process.env.LABELS;
+            const placeholder = process.env.PLACEHOLDER;
+            console.log("Placeholder:" + placeholder);
+
+            const octokit = require('@octokit/core').Octokit  
+            const mygithub = new octokit({ auth: process.env.MY_TOKEN })
+
+            let repositories = repoNames.split(",")
+            let targetLabel = encodeURIComponent(labels.split(',')[0])
+            let appendLabel = encodeURIComponent(labels.split(',')[1])
+            console.log("Repositories:" + repositories);
+            for (let r = 0; r < repositories.length; r++) {
+              console.log(`GET /repos/${process.env.OWNER}/${repositories[r]}/issues?labels=${targetLabel}`);
+              console.log(`GET /repos/${process.env.OWNER}/${repositories[r]}/issues?labels=${appendLabel}&per_page=1`);
+
+              const { data: items2add } = await mygithub.request(`GET /repos/${{env.OWNER}}/${repositories[r]}/issues?labels=${targetLabel}`, {
+              });
+
+              console.log("Items to add:" + JSON.stringify(items2add));
+
+              const { data: workMeetings } = await mygithub.request(`GET /repos/${{env.OWNER}}/${{env.REPO}}/issues?labels=${appendLabel}&per_page=1`, {
+              })
+
+              console.log("workMeetings:" + JSON.stringify(workMeetings));
+
+              try {
+                  let body = workMeetings[0].body;
+                  for (let i = 0; i < items2add.length; i++) {
+                      let url = items2add[i].html_url;
+                      let author = "@" + items2add[i].user.login;
+                      let search = parseInt(JSON.stringify(body).search(url));
+                      if (search === -1) {
+                          let startIndex = parseInt(JSON.stringify(body.indexOf(placeholder)))
+                          let json_text = JSON.stringify(body.substring(0, startIndex - 1) + `| ${url} | ${author} |\r\n` + body.substring(startIndex, body.length));
+                          body = JSON.parse(json_text);
+                      }
+                  }
+                  let template = JSON.stringify(JSON.stringify(body));
+                  let parsed = JSON.parse(JSON.parse(template));
+
+                  console.log(`PATCH /repos/${{env.OWNER}}/${{env.REPO}}/issues/${workMeetings[0].number}`);
+
+                  await mygithub.request(`PATCH /repos/${{env.OWNER}}/${{env.REPO}}/issues/${workMeetings[0].number}`, {
```

Comment:
`process.env.*` (4x on 2 lines)

Reply:
${process.env.VAR_NAME} and ${{env.VAR_NAME}} are different ways to access the same environment variable but better to use same approach to help with the code readability. I'll use ${process.env.VAR_NAME}. Thanks!